src/lib/components/compositions/dialog/dialog.svelte

```diff
@@ -41,23 +44,54 @@
 </script>
 
 <!-- Inspired by https://github.com/mvolkmann/svelte-dialog -->
-<!-- TODO: Animate modal from bottom and fade in -->
 <dialog
   id={dialogId}
   bind:this={dialogEl}
-  class={`tw_dialog-box dark:tw_dialog-box_dark md:tw_dialog-box_medium`}
+  class={`tw_dialog-box dark:tw_dialog-box_dark md:tw_dialog-box_medium ${
+    forceOpen ? '' : 'tw_dialog-box_animate'
```

Comment:
This just eliminates the animation if the dialog is forced to be open for testing purposes. In the future, we can use this turn off animation if a user has reduced motion system preference enabled.